### ISSUE TITLE
Boosted Pally, won't let me get quest for Resurrection chain

### ISSUE THREAD

**Issue by huffapuff** [dalaranwow/dalaran-wow#4355]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The quest giver for the Rez chain for pally's does not give me a quest

**Expected behaviour**: Tell us what should happen instead
Need the quest so I can learn Rez

**Steps to reproduce the problem**:

1. Problem is a constant issue, no need to reproduce. Character name - Busyflexin lv 80 Pally
2. 
3. 

**Include proofs for this behaviour**

Proof is my character, Busyflexin, no quest chain available.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
The issuetracker is for serverside and gamemechanic issues only. Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
